scripts/genappimage.sh

```diff
@@ -75,6 +76,13 @@ chmod 755 AppRun
 
 cd "$APP_BUILD_DIR" || exit # Get out of AppImage directory.
 
+# We want to be consistent, so always use arm64 over aarch64
```

Comment:
nit: if `ARCH_ORIGINAL=$ARCH` is done just before this, then we can delete the other `ARCH_ORIGINAL=...` assignments.